spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/ZuulProperties.java

```diff
@@ -357,6 +359,26 @@ public boolean isCustomSensitiveHeaders() {
 		
 	}
 
+	@Data
```

Comment:
okay, I will remove lombok annotation and add related information into the documentation about this PR soon.